layouts/partials/footer.html

```diff
@@ -5,10 +5,10 @@
 			<div class="col-md-3 widget">
 				<h3 class="widget-title">Contact</h3>
 				<div class="widget-body">
-					<p>{{ .Site.Params.phone }}<br>
-						<a href="mailto:{{ .Site.Params.email }}">{{ .Site.Params.email }}</a><br>
+					<p>{{ if .Site.Params.phone }}{{ .Site.Params.phone }}<br>{{ end }}
+					{{ if .Site.Params.email }}<a href="mailto:{{ .Site.Params.email }}">{{ .Site.Params.email }}</a><br>{{ end }}
 						<br>
-						{{ .Site.Params.address }}
+						{{ if .Site.Params.address }}{{ .Site.Params.address }}{{ end }}
```

Comment:
Not necessary at the moment, but consistent in case style changes in the future.